cli/cmd/coredump.go

```diff
@@ -17,12 +25,28 @@ func NewCoredumpCmd() *cobra.Command {
 		Use:   "pack COREDUMP",
 		Short: "pack tarantool coredump into tar.gz archive",
 		Run: func(cmd *cobra.Command, args []string) {
-			if err := coredump.Pack(args[0]); err != nil {
+			err := coredump.Pack(args[0],
+				coredumpPackExecutable,
+				coredumpPackOutputDirectory,
+				coredumpPackPID,
+				coredumpPackTime,
+			)
+			if err != nil {
 				util.HandleCmdErr(cmd, err)
 			}
 		},
 		Args: cobra.ExactArgs(1),
 	}
+	packCmd.Flags().StringVarP(&coredumpPackExecutable, "executable", "e", "",
+		"Tarantool executable path")
+	packCmd.Flags().StringVarP(&coredumpPackOutputDirectory, "directory", "d", "",
+		"Directory the resulting archive is created in")
+	packCmd.Flags().StringVarP(&coredumpPackTime, "time", "t", "",
+		"Time of dump, expressed as seconds since the Epoch (1970-01-01 00:00 UTC)")
+	packCmd.Flags().UintVarP(&coredumpPackPID, "pid", "p", 0,
+		"PID of the dumped process, as seen in the PID namespace in which\n"+
+			"the given process resides (see %p in core(5) for more info). This flag\n"+
+			"is to be used when tt is used as kernel.core_pattern pipeline script")
```

Comment:
```suggestion
		"the given process resides (see %p in core(5) for more info). This flag\n"+
		"is to be used when tt is used as kernel.core_pattern pipeline script")
```

Reply:
The command:
`goimports -w cli/cmd/coredump.go`

may help to format the code and resolve all problems in formatting.

Reply:
@patapenka-alexey , this is how go decorates multiline string pattern (I suppose the indentation is used to point out that this is continuation of the string.
@oleg-jukovec , it seems vscode extension does this formatting for me. Thanks for info, anyway.